app/view/grid/Grid.js

```diff
@@ -15,6 +15,8 @@ Ext.define('CpsiMapview.view.grid.Grid', {
         'CpsiMapview.view.button.FeatureSelectionButton'
     ],
 
+    mixins: ['CpsiMapview.util.ColumnMenuOrderMixin'],
```

Comment:
One question on this approach - if a grid is a subclass of `CpsiMapview.view.grid.Grid` and defines its own `mixins` property does it override the `['CpsiMapview.util.ColumnMenuOrderMixin']` or append to it? Not sure if we have any grids where this is the case at the moment

Reply:
Mixins get inherited like other members so in this case, mixins defined in the subclass would exist along side `CpsiMapview.util.ColumnMenuOrderMixin`